vslib/inc/SwitchBCM56850.h

```diff
@@ -24,6 +24,9 @@ namespace saivs
 
         protected:
 
+            sai_status_t create_cpu_qos_queues(
```

Comment:
since this could be the case also in other platforms maybe this could be virtual functions override and not implemented function in base class SwitchStateBase like create_qos_queues_per_port?/